docs/reference/ccr/upgrading.asciidoc

```diff
@@ -1,49 +1,67 @@
 [role="xpack"]
 [testenv="platinum"]
 [[ccr-upgrading]]
-=== Upgrading clusters
+=== Upgrading clusters using {ccr}
+++++
+<titleabbrev>Upgrading clusters</titleabbrev>
+++++
 
 Clusters that are actively using {ccr} require a careful approach to upgrades.
-Otherwise index following may fail during a rolling upgrade, because of the
-following reasons:
+The following conditions could cause index following to fail during rolling
+upgrades:
 
-* If a new index setting or mapping type is replicated from an upgraded cluster
-  to a non-upgraded cluster then the non-upgraded cluster will reject that and
-  will fail index following.
-* Lucene is not forwards compatible and when index following is falling back to
-  file based recovery then a node in a non-upgraded cluster will reject index
-  files from a newer Lucene version compared to what it is using.
-
-Rolling upgrading clusters with {ccr} is different in case of uni-directional
-index following and bi-directional index following.
+* Clusters that have not yet been upgraded will reject new index settings or
+mapping types that are replicated from an upgraded cluster.
+* Nodes in a cluster that has not been upgraded will reject index files from a
+node in an upgraded cluster when index following tries to fall back to
+file-based recovery. This limitation is due to Lucene not being forward
+compatible.
 
+The approach to running a rolling upgrade on clusters where {ccr} is
+enabled differs based on uni-directional and bi-directional index following.
 
+[[ccr-uni-directional-upgrade]]
 ==== Uni-directional index following
 
-In a uni-directional setup between two clusters, one cluster contains only
-leader indices, and the other cluster contains only follower indices following
-indices in the first cluster.
+In a uni-directional configuration, one cluster contains only
```

Comment:
It's probably worth mentioning up front that you can have bi-directional replication between clusters.